module/behaviour/strategy/KeyboardWalk/src/KeyboardWalk.cpp

```diff
@@ -50,7 +49,7 @@ namespace module::behaviour::strategy {
         initscr();
 
         // Initialise colours
-        if (has_colors() == TRUE) {
+        if (static_cast<int>(has_colors()) == TRUE) {
```

Comment:
This seems incorrect. I think clang-tidy has correctly deduced that TRUE is defined somewhere as `#define TRUE 1` and is, therefore, an int. And `has_colors()` is returning a bool so it has correctly decided that comparing a bool to an integer is a bad thing. Where it went wrong was rather than changing `TRUE` to `true` so it's bool to bool comparison, it changed `has_colors` to an int